customization-guide.md

```diff
@@ -0,0 +1,106 @@
+- [**ims_dataset**](roles/ims_dataset) - creates and deletes IMS definition data sets and libraries
+- [**ims_dbrc**](roles/ims_dbrc) - sets up DBRC defaults and prepares DBRC
+- [**ims_exit**](roles/ims_exit) - prepares security and connection exit for IMS
+- [**ims_gen**](roles/ims_gen) - prepares DBDGEN, PSBGEN, and ACBGEN
+- [**ims_iefjobs**](roles/ims_iefjobs) - creates IEF jobs
+- [**ims_initialize**](roles/ims_initialize) - reserves ICON ports and sends PROCLIB templates to zOS
+- [**ims_online_change**](roles/ims_online_change) - enables the online change utility
+- [**ims_operations_manager**](roles/ims_operations_manager) - defines, starts, stops, and queries operations manager
+- [**ims_proclib**](roles/ims_proclib) - defines BPE configuration and copies PROCLIB
+- [**ims_racf**](roles/ims_racf) - prepares RACF security for IMS
+- [**ims_region**](roles/ims_region) - starts IMS regions
+- [**ims_resource_manager**](roles/ims_resource_manager) - defines, starts, stops and queries RM
+- [**ims_structured_call_interface**](roles/ims_structured_call_interface) - defines, starts, stops, and queries SCI
+- [**ims_sysdef**](roles/ims_sysdef) - prepares system definition data set
+- [**ims_tls**](roles/ims_tls) - configure AT-TLS ports
+- [**install-bzip2**](roles/install-bzip2) - copies and installs utility zip file on zOS
+- [**monitor_ims**](roles/monitor_ims) - checks to see if IMS is up or down.
+- [**provision_ims**](roles/provision_ims) - this is the main role to provision IMS.  It utilizes other roles to allocate datasets, prepare datasets and start all IMS services
+- [**save-templates-to-datasets**](roles/save-templates-to-datasets) - copies templates from USS to zOS data sets
+- [**send-template**](roles/send-template) - sends a template file from the local host to zOS
+- [**send-templates**](roles/send-templates)  - sends multiple templates in a directory from local host to zOS
+- [**set_up_run_environment**](roles/set_up_run_environment)  - setting up the initial environment for playbook execution
+- [**stop_jmp**](roles/stop_jmp) - stop JMP (Java) region
+- [**stop_mpp**](roles/stop_mmp) - stop MPP region
+- [**submit-rexx**](roles/submit-rexx) - runs a REXX script on zOS
+
+
+Inside each role:
+
+* `./tasks` contains tasks that can be performed by the role.
+  Sub-folder names describe the use of the contained files.
+* `./tasks/main.yml` contains the default tasks performed by the role.
+
+
+## Requirements
+* IBM z/OS core collection 1.5.0
+* IBM z/OS IMS collection 1.2.0
+* IBM® Wazi Sandbox 2.4 or IBM® Extended z/OS® ADCD for Z Development and Test Environment built upon the general release of ADCD z/OS® V2R5 December Edition of 2022 or later
+
+## Getting Started
+
+Consult the [Operator Collection SDK](https://github.com/IBM/operator-collection-sdk) for basic concepts and tutorials before modifying this collection for your own use. 
```

Comment:
We can move Getting Started section higher up in the guide. For example, it can go after the first section and before operator customization. Playbook customization/summary can come in after that.